portability-spi-cloud/src/main/java/org/dataportabilityproject/spi/cloud/types/PortabilityJob.java

```diff
@@ -12,76 +14,134 @@
  */
 public class PortabilityJob extends EntityType {
 
-    /**
-     * The job states.
-     */
-    public enum State {
-        NEW, COMPLETE, ERROR
+  /**
+   * The job states.
+   */
+  public enum State {
+    NEW, COMPLETE, ERROR
+  }
+
+  @JsonProperty
+  private State state = State.NEW;
+
+  @JsonProperty
+  private String exportService;
+
+  @JsonProperty
+  private String importService;
+
+  @JsonProperty
+  private String transferDataType;   // REVIEW: replace old PortableDataType since the latter is an enum and not extensible?
```

Comment:
another one left in here now :P 
fine with my but just letting you know in case you meant to get rid of it